openmc/settings.py

```diff
@@ -1692,6 +1717,14 @@ def _weight_windows_from_xml_element(self, root, meshes=None):
         if meshes is not None and self.weight_windows:
             meshes.update({ww.mesh.id: ww.mesh for ww in self.weight_windows})
 
+    def _weight_window_checkpoints_from_xml_element(self, root):
+        elem = root.find('weight_window_checkpoints')
+        if elem is not None:
+            return
+        for key in ('collision', 'surface'):
+            value = get_text(elem, key)
+            if value is not None:
+                self.weight_window_checkpoints[key] = value
```

Comment:
```suggestion
                self.weight_window_checkpoints[key] = value

```

Reply:
Are there any differences?

Reply:
This adds a line between the end of this function and the one below.